Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-single-multiple-answer/questions-single-multiple-answer.html

```diff
@@ -16,10 +16,10 @@ <h5 *ngIf="!isSingleAnswerQuestion">Multiple Answer Question</h5>
                 <div class="form-group answer-choices" *ngIf="isSingleAnswerQuestion">
                     <md-radio-group [(ngModel)]="indexOfOptionSelected" name="first">
                         <md-radio-button value="{{i}}" [disabled]="option.option === undefined || option.option === ''" *ngFor="let option of singleMultipleAnswerQuestion.singleMultipleAnswerQuestion.singleMultipleAnswerQuestionOption;let i = index" [attr.data-index]="i">
-                            <input type="text" class="form-control" placeholder="Answer Choice {{i+1}}" id="choice" #choice="ngModel" [(ngModel)]="option.option" name="optionDetail_{{i}}" required />
+                            <input type="text" class="form-control" placeholder="Answer Choice {{i+1}}" #choice="ngModel" [(ngModel)]="option.option" name="optionDetail_{{option.id}}" required />
                             <em class="material-icons" (click)="removeOption(i)" *ngIf="!isClose">close</em>
                             <div class="errors-container">
-                                <span *ngIf="choice.invalid && choice.touched" class="error-msg mt9 ml10">This field is required</span>
+                                <span *ngIf="choice.touched && choice.invalid " class="error-msg mt9 ml10">This field is required</span>
```

Comment:
this change does not make any sense. There is and condition so until and unless both are true nothing will happen. so there is no meaning of altering condition.